src/projects/TodoList/index.js

```diff
@@ -61,6 +83,23 @@ class TodoList extends React.Component {
     alteredTask.editingTask = value;
     this.setState({ todos: this.state.todos });
   }
+  toggleCheckBox(i) {
+    let alteredCheckBox = this.state.categories[i];
+    alteredCheckBox.checked = !alteredCheckBox.checked;
+    this.setState({ categories: this.state.categories });
+  }
+  addCategory() {
+    const largestCategoryId = this.state.categories[(this.state.categories).length-1].id
+    const newCategoryColour = "rgb(" + this.state.newCategoryRed + "," + this.state.newCategoryGreen + "," + this.state.newCategoryBlue + ")";
```

Comment:
You can use the backticks `` here to do a string concatenation rather than adding a load of strings together like this
```js
const string1 = 'hello';
const string2 = 'there';
const longString = "string1 + " " + "string 2" // "hello there"
```
can be written as
```js
const string1 = 'hello';
const string2 = 'there';
const longString = `${string1} ${string2}`
```